components/elm/src/biogeophys/EnergyFluxType.F90

```diff
@@ -380,8 +392,171 @@ subroutine Restart(this, bounds, ncid, flag)
          long_name='', units='', &
          interpinic_flag='interp', readvar=readvar, data=this%btran2_patch) 
 
+    call restartvar(ncid=ncid, flag=flag, varname='BTRAN_MIN', xtype=ncd_double,  &
+         dim1name='pft', &
+         long_name='daily minimum of transpiration wetness factor', units='', &
+         interpinic_flag='interp', readvar=readvar, data=this%btran_min_patch) 
+
+    call restartvar(ncid=ncid, flag=flag, varname='BTRAN_MIN_INST', xtype=ncd_double,  &
+         dim1name='pft', &
+         long_name='instantaneous daily minimum of transpiration wetness factor', units='', &
+         interpinic_flag='interp', readvar=readvar, data=this%btran_min_inst_patch) 
+
     call this%eflx_dynbal_dribbler%Restart(bounds, ncid, flag)
 
   end subroutine Restart
 
+
+  !-----------------------------------------------------------------------
+  subroutine InitAccBuffer (this, bounds)
+    !
+    ! Original source of this sub-routine:
+    ! https://github.com/ESCOMP/CTSM
+    !
+    ! !DESCRIPTION:
+    ! Initialize accumulation buffer for all required module accumulated fields
+    ! This routine set defaults values that are then overwritten by the
+    ! restart file for restart or branch runs
+    ! Each interval and accumulation type is unique to each field processed.
+    ! Routine [initAccBuffer] defines the fields to be processed
+    ! and the type of accumulation. 
+    ! Routine [updateAccVars] does the actual accumulation for a given field.
+    ! Fields are accumulated by calls to subroutine [update_accum_field]. 
+    ! To accumulate a field, it must first be defined in subroutine [initAccVars] 
+    ! and then accumulated by calls to [updateAccVars].
+    ! Four types of accumulations are possible:
+    !   o average over time interval
+    !   o running mean over time interval
+    !   o running accumulation over time interval
+    ! Time average fields are only valid at the end of the averaging interval.
+    ! Running means are valid once the length of the simulation exceeds the
+    ! averaging interval. Accumulated fields are continuously accumulated.
+    ! The trigger value "-99999." resets the accumulation to zero.
+    !
+    ! !USES 
+    use accumulMod       , only : init_accum_field
+    use elm_time_manager , only : get_step_size_real
+    !
+    ! !ARGUMENTS:
+    class(energyflux_type) :: this
+    type(bounds_type), intent(in) :: bounds
+    !
+    ! !LOCAL VARIABLES: 
+    real(r8) :: dtime
+    integer, parameter :: not_used = huge(1)
+    !---------------------------------------------------------------------
+
+    dtime = get_step_size_real()
+
+    call init_accum_field(name='BTRANAV', units='-', &
```

Comment:
Similarly, how about `BTRANAV` to `BTRAN_AVG`?

Reply:
I tried to update this name, but the CI tests all failed. It turns out that the subroutines in `accumulMod.F90` expect variable names of up to 8 characters. I reverted it back to a shorter name (`BTRANAVG`, which has exactly 8 characters).